aspnetcore/fundamentals/logging/index.md

```diff
@@ -1633,7 +1638,7 @@ Some third-party frameworks can perform [semantic logging, also known as structu
 Using a third-party framework is similar to using one of the built-in providers:
 
 1. Add a NuGet package to your project.
-1. Call an `ILoggerFactory` extension method provided by the logging framework.
+1. Call an `ILoggerFactory` or `ILoggingBuilder` extension method provided by the logging framework.
```

Comment:
Since 2.0, the recommendation is to provide ILoggingBuilder extension methods. This change better reflects that.